plugin/trino-atop/src/test/java/io/trino/plugin/atop/TestingAtopFactory.java

```diff
@@ -51,7 +52,7 @@ public Atop create(AtopTable table, ZonedDateTime date)
         private TestingAtop(InputStream dataStream, ZonedDateTime date)
         {
             this.date = date;
-            this.reader = new BufferedReader(new InputStreamReader(dataStream));
+            this.reader = new BufferedReader(new InputStreamReader(dataStream, UTF_8));
```

Comment:
is it reading a test resource? then utf-8 is ok

Reply:
Yes, the stream is read from `src/test/resources`:

```java
InputStream data = TestingAtopFactory.class.getResourceAsStream(table.name().toLowerCase(Locale.ENGLISH) + ".txt");
requireNonNull(data, format("No data found for %s", table));
return new TestingAtop(data, date);
```

There are two files, for which even US-ASCII would be fine.

Reply:
> Yes, the stream is read from `src/test/resources`:

test resources are guaranteed to be utf-8